packages/image_picker/image_picker/example/ios/Flutter/AppFrameworkInfo.plist

```diff
@@ -25,6 +25,6 @@
     <string>arm64</string>
   </array>
   <key>MinimumOSVersion</key>
-  <string>8.0</string>
+  <string>9.0</string>
```

Comment:
This was done automatically via https://github.com/flutter/flutter/pull/85174 by just `flutter build`ing the example project.